src/software/ai/hl/stp/play/shoot_or_pass_play_test.cpp

```diff
@@ -40,7 +40,7 @@ TEST_F(ShootOrPassPlayTest, test_shoot_or_pass_play)
         // TODO: Implement proper validation
         // https://github.com/UBC-Thunderbots/Software/issues/1971
         [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
-            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(9.5))
+            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(24.5))
             {
                 yield("Timestamp not at 9.5s");
```

Comment:
I think you forgot to change the 9.5 to 24.5, since you changed it above

Reply:
good catches, fixed